configs/example.config.yaml

```diff
@@ -2,7 +2,9 @@ app:
   log_level: info
 
 client_urls:
-  binancedex: "https://dex-atlantic.binance.org"
+  binance:
+    explorer: https://explorer.binance.org
```

Comment:
is there a way to ditch `explorer` endpoint and use just `dex?

Reply:
i will figure out

Reply:
there's no way